### PR TITLE
Created Reagents and New Legendary Req Suffixes

### PR DESCRIPTION
**Pull request by JukeDragh**

### CODE REVIEW

File: strings/reagents.txt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+goat cheese
+guardian spirit
+honeycomb
+hormel
+liquid fate
+liquid grind
+loc of black hair
+magical thing viscera
+norkian small fish
+old thong fiber
```
Comment (seiyria) [inline strings/reagents.txt]: ok so this is funny but

File: strings/reagents.txt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+six-leaf
+something off on the ground
+tears of creation
+tears of providence
+testicle of newt
+vector dust
+venom of gorgon snakes
+void shards
+witch's boil
+[redacted]
```
Comment (seiyria) [inline strings/reagents.txt:48]: I fuckin lost it when I got here

**Comment by JukeDragh** [comment]
I hope my laughs will be found in the players soon enough

**Comment by JukeDragh** [comment]
your*

**Comment by JukeDragh** [comment]
Btw, I would use $dict:reagents$ to pick one of these right?

**Comment by seiyria** [comment]
That should work.

**Pull request merged**